src/lib/state.ts

```diff
@@ -36,17 +36,23 @@ function getNodeValues(node: ZWaveNode): ValueState[] {
   );
 }
 
-export const dumpValue = (
-  node: ZWaveNode,
-  valueId: TranslatedValueID
-): ValueState => {
-  const valueState = valueId as ValueState;
-  valueState.metadata = node.getValueMetadata(valueId);
-  valueState.value = node.getValue(valueId);
+export const dumpValue = (node: ZWaveNode, valueArgs: any): ValueState => {
```

Comment:
Did you test this? It seems like we're changing the signature and miss out on typing ?

Reply:
Yes, I did test this. valueArgs can be either ValueChangedArgs, ValueAddedArgs etc. etc so it was either a lot of OR's types or abusing any.

Reply:
adjusted